.gitignore

```diff
@@ -4,7 +4,6 @@ bitlbee
 user-guide.txt
 user-guide.html
 help.txt
-debian
```

Comment:
Hmm ok that's probably a matter of taste ;) I usually don't expect there to be any auto-generated stuff left after building with gbp or debuild so if git status shows me any untracked files in debian/ I know either something is broken or I forgot to add something I changed - if the dir is ignored, I could miss something important.